dev/build_libhdfs3.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
can you please also help to document this in the velox backend doc?

Reply:
@zhouyuan Yes. Added the document.